### PR TITLE
Step2_1 into version2 branch

### PR DESCRIPTION
**Pull request by Sergei-Nerobeev**
Fixed branch.

### CODE REVIEW

File: .idea/sonarlint/issuestore/index.pb
Comment (danikirillov) [inline .idea/sonarlint/issuestore/index.pb:1]: remove this files with gitignore

File: src/hu/nero/toyota/detail/Transmission.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package hu.nero.toyota.detail;
 
 public enum Transmission {
-  AUTOMATE,ROBOT, MECHANICAL;
+  AUTOMATE, ROBOT, MECHANICAL, SOME_VOLUME;
```
Comment (danikirillov) [inline src/hu/nero/toyota/detail/Transmission.java:4]: some_volume for what?

**Review by danikirillov — APPROVED** [review]
good job👍

File: src/hu/nero/toyota/detail/Transmission.java (hunk 1 above)
Comment (Sergei-Nerobeev) [reply src/hu/nero/toyota/detail/Transmission.java:4]: Done.

File: src/hu/nero/toyota/factory/DetailFactory.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    public DetailFactory(Country countryAssembly) {
+        this.countryAssembly = countryAssembly;
+    }
+
+    public Wheel createWheel(WheelRadius wheelRadius) {
+        return new Wheel(wheelRadius);
+    }
+
+    public Transmission createTransmission() {
+        return Transmission.SOME_VOLUME;
```
[the remark this answers is not in the document]
Comment (Sergei-Nerobeev) [reply src/hu/nero/toyota/factory/DetailFactory.java:17]: Deleted.

**Pull request merged**